src/main/java/org/folio/consortia/service/impl/TenantManagerImpl.java

```diff
@@ -147,15 +152,27 @@ public void delete(UUID consortiumId, String tenantId, TenantDeleteRequest tenan
   }
 
   @Override
-  public void createIdentityProvider(String memberTenantId) {
-    keycloakService.createIdentityProvider(folioExecutionContext.getTenantId(), memberTenantId);
+  public void createIdentityProvider(String memberTenantId, IdentityProviderCreateRequest idpCreateRequest) {
+    if (idpCreateRequest.getCreateProvider()) {
+      keycloakService.createIdentityProvider(folioExecutionContext.getTenantId(), memberTenantId);
+    }
+    if (idpCreateRequest.getMigrateUsers()) {
```

Comment:
for request contract I think it is okay to remain migrateUsers, because it is relates to IdentityProviderCreateRequest, so contract we can remain as it, just rename services